test/framework/utilities.py

```diff
@@ -51,6 +51,30 @@
 from easybuild.tools.filetools import mkdir, read_file
 from easybuild.tools.module_naming_scheme import GENERAL_CLASS
 from easybuild.tools.modules import modules_tool
+from easybuild.tools.options import EasyBuildOptions
+
+
+# make sure tests are robust against any non-default configuration settings;
+# involves ignoring any existing configuration files that are picked up, and cleaning the environment
+# this is tackled here rather than in suite.py, to make sure this is also done when test modules are ran separately
```

Comment:
why not add in the enhancedtestcase setup method
```python
import easybuild.tools.options
easybuild.tools.options.CONFIG_ENV_VAR_PREFIX = 'EASYBUILD_TEST'
easybuild.tools.options.EasyBuildOptions = False
```
(and restore in the cleanup method?)